src/helpers/DataRequirementHelpers.ts

```diff
@@ -69,6 +77,28 @@ export function generateDetailedDateFilter(filter: DuringFilter): R4.IDataRequir
   };
 }
 
+/**
+ * Map a filter into a FHIR DataRequirement valueFilter extension
+ *
+ * @param filter the filter to map
+ * @returns extension for the valueFilter list of dataRequirement
+ */
+export function generateDetailedValueFilter(filter: Filter): R4.IExtension {
+  if (filter.type === 'notnull') {
+    const notnullFilter = filter as NotNullFilter;
+    return {
+      url: 'http://example.com/dr-value',
```

Comment:
Just double checking, these are place holder URL's until we have a defined one in our Profile, right?